plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -71,7 +71,8 @@ def change_location_tag(primary_xml_snippet, relpath):
     end_portion = string_to_unicode(primary_xml_snippet[end_index:])
     location = string_to_unicode("""<location href="%s"/>""" % (
         file_utils.make_packages_relative_path(relpath)))
-    return first_portion + location + end_portion
+    modified_xml_snippet = first_portion + location + end_portion
+    return modified_xml_snippet.encode('utf-8')
```

Comment:
This one was for consistency since this `change_location_tag` was the only source of unicode. The rest of metadata coming from parsing during upload or sync was in utf-8. I was trying to avoid `if unicode: encode into utf-8`.